backend/unit/unit.go

```diff
@@ -0,0 +1,40 @@
+package unit
+
+import (
+	"errors"
+	"strconv"
+	"time"
+)
+
+const (
+	oneDay  = time.Hour * 24
+	oneWeak = oneDay * 7
+)
+
+// ParseDuration reads a human-readable duration and returns duration in seconds
```

Comment:
I guess we won't support `ms` for now. How about adding a TODO so that we can think about extending to `ms` or adding other units, such as storage (B, KB, MB, GB, TB, PB) and `temperature ( C, F )`. This also implies that we may include other data structures for `temperature` or `storage` representation.